requirements.txt

```diff
@@ -6,3 +6,4 @@ uwsgi==2.0.10
 psycopg2==2.6
 tox==1.9.2
 static3==0.5.1
+git+https://github.com/mitodl/xbundle.git@7799a0957c502103fb030fc0b16c64204f551c75#egg=xbundle
```

Comment:
I just thought of this, but I think we should add commented sections for each app so we can pull apart deps.  Both here, and in apt.txt as well.  So far everything in there has been core, but this and the xml libs in apt, would be either import app deps with a comment stating such.